troubleshooting/index.md

```diff
@@ -2,105 +2,78 @@
 
 This document provides troubleshooting information for services and functionality provided by CSM.
 
-**Topics**
+### Known Issues
  * [Known Issues](#known-issues)
-    * [Hardware Discovery](#known-issues-hardware-discovery)
-    * [initrd.img.xz not found](#initrd-not-found)
-    * [SAT/HSM/CAPMC Component Power State Mismatch](known_issues/component_power_state_mismatch.md)
-    * [Wait_for_unbound or `cray-dns-unbound-manager` hangs](known_issues/wait_for_unbound_hang.md)
-
- * Kubernetes troubleshooting
-    * [General Kubernetes Commands for Troubleshooting](./kubernetes/Kubernetes_Troubleshooting_Information.md)
-    * [Kubernetes Log File Locations](./kubernetes/Kubernetes_Log_File_Locations.md)
-    * [Troubleshoot Liveliness or Readiness Probe Failures](./kubernetes/Troubleshoot_Liveliness_Readiness_Probe_Failures.md)
-    * [Troubleshoot Unresponsive kubectl Commands](./kubernetes/Troubleshoot_Unresponsive_kubectl_Commands.md)
-    * [Troubleshoot Kubernetes Node NotReady](./kubernetes/Troubleshoot_Kubernetes_Node_NotReady.md)
-
+    * [Hardware Discovery](known_issues/Hardware_Discovery.md)
+    * [SAT/HSM/CAPMC Component Power State Mismatch](https://github.com/Cray-HPE/docs-csm/blob/CASMINS-4311/troubleshooting/known_issues/component_power_state_mismatch.md)
```

Comment:
I don't think these should be linking to the CASMINS-4311 branch.

Reply:
I think these should all be relative paths so they stay with the content in the same branch.

Reply:
Fixed